catalog/src/test/java/it/eng/catalog/service/ProxyAPIServiceTest.java

```diff
@@ -57,22 +62,39 @@ void getFormatsFromDataset_fail() {
 		when(genericApiResponse.isSuccess()).thenReturn(false);
 		
 		assertThrows(CatalogErrorAPIException.class, 
-				() -> service.getFormatsFromDataset(CatalogMockObjectUtil.DATASET_ID, forwardTo));
+				() -> service.getFormatsFromDataset(CatalogMockObjectUtil.DATASET_ID, FORWARD_TO));
 	}
 
 	@Test
 	@DisplayName("Fetch proxy catalog")
 	void getCatalog() {
 		mockCatalogCall();
-		Catalog catalog = service.getCatalog(forwardTo);
+		Catalog catalog = service.getCatalog(FORWARD_TO);
 		assertNotNull(catalog);
 	}
 
 	private void mockCatalogCall() {
+		Catalog CATALOG = Catalog.Builder.newInstance()
```

Comment:
This is the part that you stated was causing error? Had to create new catalog and not use that global one?